public/app/features/transformers/docs/content.ts

```diff
@@ -600,41 +625,55 @@ export const transformationDocsContent: TransformationDocsContentType = {
   | server 1                | 82   |          |
   | server 2                | 88.6 |          |
   | server 3                |      | 59.6     |
+
+  Utilize this transformation to construct a matrix by specifying fields from your query results. The matrix output reflects the relationships between the unique values in these fields. This helps you present complex relationships in a clear and structured matrix format.
   `;
     },
   },
   heatmap: {
     name: 'Create heatmap',
     getHelperDocs: function () {
       return `
-  Use this transformation to prepare histogram data to be visualized over time. Similar to the Heatmap panel, this transformation allows you to convert histogram metrics to buckets over time.
+  Use this transformation to prepare histogram data for visualizing trends over time. Similar to the Heatmap panel, this transformation converts histogram metrics into temporal buckets.
```

Comment:
We're moving towards calling visualization types, _visualizations_ rather than _panels_. Because that's what they are 😉 And unless we're specifically referring to the UI, they don't need to be capitalized anymore.
```suggestion
  Use this transformation to prepare histogram data for visualizing trends over time. Similar to the heatmap visualization, this transformation converts histogram metrics into temporal buckets.
```